### PR TITLE
LP-239 Add the limited partner choice page

### PR DESCRIPTION
**Pull request by markpit**
### JIRA link
https://companieshouse.atlassian.net/browse/LP-239

### Change description
Add the limited partner choice page

### Work checklist

- [x] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by HatiDev — APPROVED** [review]
LGTM

File: src/presentation/test/integration/registration/limited-partner-choice.test.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  beforeEach(() => {
+    setLocalesEnabled(false);
+  });
+
+  const setLocalesEnabled = (bool: boolean) => {
+    jest.spyOn(config, "isLocalesEnabled").mockReturnValue(bool);
+    LocalesService.getInstance().enabled = bool;
+  };
+
+  const insertIdsInUrl = (url: string, transactionId: string, submissionId: string) => {
```
Comment (lduranteau) [inline src/presentation/test/integration/registration/limited-partner-choice.test.ts:24]: there is already a method accessible from the dev dependencies
look at the email.test.ts

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by lduranteau — APPROVED** [review]

**Pull request merged**